packages/cli/src/commands/validate/validate.test.ts

```diff
@@ -28,9 +28,21 @@ describe('validate', () => {
       .invoke(['validate', './packages/cli/src/commands/validate/invalid.json'])
 
     logs.should.contain(
-      'Cannot find supported Swagger/OpenAPI version in specification, version must be a string.',
+      'Can’t find supported Swagger/OpenAPI version in specification, version must be a string.',
```

Comment:
should we make this message friendlier in the command file?

```suggestion
      'Unable to locate a supported Swagger/OpenAPI version in specification. Ensure that the version is a string.',
```

Reply:
Love it! But it’s coming from the parser, I’ll change it there.